compose.dev.yml

```diff
@@ -0,0 +1,101 @@
+version: "3.6"
```

Comment:
I assume we would have Dockerfile.dev and Dockerfile.prod in coming time. We can still leave that docker-compose file in case if someone is not able to setup dev environment via docker (generally in case of Windows host machine without WSL2). Current PR only works for linux and WSL based systems.